frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -322,32 +333,27 @@ const Map: React.FC<MapProps> = ({
     }
   };
 
-  const handleBounds = (e: any) => {
+  const handleMoveEnd = (e: LeafletEvent) => {
     const boundsData: LatLngBounds = e.target.getBounds();
     if (!isEqual(boundsData.getNorthEast(), boundsData.getSouthWest())) {
       setBounds(boundsData);
     }
   };
 
-  const [infoOpen, setInfoOpen] = React.useState(false);
-  const [layersOpen, setLayersOpen] = React.useState(false);
-  const displayMessage = (
-    <p>
-      You might have clicked outside of a parcel boundary or you do not have access to the parcel
-      layer yet. You can zoom into the map to ensure that you are actually clicking within the
-      boundaries of a parcel. Please contact{' '}
-      <a href="mailto:CITZ_RPD_IMIT_HELP@gov.bc.ca">CITZ_RPD_IMIT_HELP@gov.bc.ca</a> if you are
-      still unable to access the parcel layer details.
-    </p>
-  );
+  const handleResize = () => {
+    // The map has changed and needs to be redrawn and possibly zoomed and centered.
+    mapRef.current?.leafletElement.invalidateSize();
+    const z = mapRef.current?.leafletElement.getZoom();
+    if (zoom !== z) mapRef.current?.leafletElement.setView(center, zoom);
+  };
+
   return (
-    <ReactResizeDetector handleWidth>
-      {({ width }: any) => {
-        setMapWidth(width);
+    <ReactResizeDetector handleWidth refreshMode="throttle" onResize={handleResize}>
+      {() => {
```

Comment:
Is this necessary? Can't `<Container>` just be a child of `<ReactResizeDetector>`?

Reply:
I did try this and it resulted in an error that I was unable to resolved.  The error was something to do with how ReactResizeDetector uses the `targetRef` attribute.  Something is bad with the library if you ask me.  So I've left it the way it was originally implemented.